src/index.d.ts

```diff
@@ -754,16 +768,16 @@ declare namespace RamdaAdjunct {
          * from fromIndex (inclusive).
          * Dispatches to the slice method of the third argument, if present.
          */
-        sliceFrom<T>(fromIndex: number, list: string|T[]): string|T[];
-        sliceFrom(fromIndex: number): <T>(list: string|T[]) => string|T[];
+        sliceFrom<T>(fromIndex: number, list: string | T[]): string | T[];
```

Comment:
Just for info: All these changes in typings are related to new version of tslint ?

Reply:
Yes could be part of this PR. The question was rather if  the tslint rules changes to create these changes or did you do them manually ?

Reply:
No. I haven't made any changes outside of the immediate scope of the PR.